### PR TITLE
Decode a Morse code message

### PR DESCRIPTION
**Pull request by medaminedev66**
- Create a method `decode_char` to decode a Morse code character, it takes a string parameter and returns the corresponding character in uppercase.

- Create a method `decode_word` to decode an entire word in Morse code, it takes a string parameter and returns the string representation.

- Create a method `decode_sentence` to decode the entire message in Morse code, it takes a string parameter and returns the string representation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# morse-code
+# morse-code
```
Comment (EricMbouwe) [inline README.md:1]: - [ ] Make sure that you documented your work in a [professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md). Kindly add a more descriptive readme

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
### Hi STUDENT,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight!
- Good PR ✔️
- Good commits ✔️
- Working script ✔️

# Required Changes ♻️ ⛔️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (henatan99) [reply README.md:1]: This issue has been well addressed. Nice job! :clap:

**Review by henatan99 — APPROVED** [review]
Hi @medaminedev66 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**